packages/connect-webextension/src/proxy/index.ts

```diff
@@ -59,6 +59,16 @@ const init = (settings: Partial<ConnectSettings> = {}): Promise<void> => {
         }
     });
 
+    const reconnect = () => {
+        // By connecting again we keep the service worker active.
+        cancel();
+        _channel = null;
+        init(settings);
+    };
+
+    _channel.port.onDisconnect.removeListener(reconnect);
+    _channel.port.onDisconnect.addListener(reconnect);
+
```

Comment:
I am still thinking in a different ways of using this way but for now this works for me.

Reply:
@martykan Let me know if this works on your machine.

Reply:
Yes, this looks very good! 👍

Reply:
@martykan what do you think should we proceed with this like it is?
I was thinking in other approaches but don't want to over complicate it.

Reply:
It seems to work and doesn't require any extra configuration so this is probably the best

Reply:
:+1:  Let's approve and merge it then!